docs/data/toolpad/core/components/dashboard-layout/DashboardLayoutBasic.js

```diff
@@ -99,22 +101,32 @@ function DashboardLayoutBasic(props) {
 
   const router = useDemoRouter('/dashboard');
 
-  // Remove this const when copying and pasting into your project.
+  // Remove these variables when copying and pasting into your project.
   const demoWindow = window !== undefined ? window() : undefined;
+  const demoEmotionCache = React.useMemo(
```

Comment:
Adding this in each demo that uses a whole `AppProvider` would solve this issue.